process-engine/bpmn-studio
# ♻️ Improve Diagram Changed Handling

## Changes

1. Only Show Change Notification if Diagram Gets Changed Outside the Studio
2. Reload Diagram if Diagram Is Unchanged in Studio And Changed Outside of the Studio

## Issues

Closes #1770 

PR: #1786

## How to test the changes

- Open a diagram in the BPMN Studio (without changing it!)
- Open it in another application (e.g. another version of the BPMN Studio)
- **Notice that no notification gets displayed**
- Save it in another application (without changing it!)
- **Notice that no notification gets displayed**
- Change & save the diagram in another application
- **Notice that the diagram gets updated in the BPMN Studio**
- In another application, change the diagram back to its previous state and save it
- **Notice that the diagram gets updated in the BPMN Studio**
- Change the diagram in the BPMN Studio (without saving it!)
- Change & save the diagram in another Application
- **Notice that a notification gets displayed in the BPMN Studio**
